docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc

```diff
@@ -267,14 +256,16 @@ novadb_svc_records=$(oc exec openstack-cell1-galera-0 -c galera -- mysql -rs -ur
 diff -Z <(echo $novadb_svc_records) <(echo $PULL_OPENSTACK_CONFIGURATION_NOVA_COMPUTE_HOSTNAMES)
 ----
 
-. During the pre/post checks the pod `mariadb-client` might have returned a pod security warning
-related to the `restricted:latest` security context constraint. This is due to default security
-context constraints and will not prevent pod creation by the admission controller. You'll see a
-warning for the short-lived pod but it will not interfere with functionality.
-. Delete the `mariadb-data` pod and `mariadb-copy-data` persistent volume claim with databases backup (consider making a snapshot of it, before deleting)
+. Delete the `mariadb-data` pod and `mariadb-copy-data` persistent volume claim with databases backup:
++
+[NOTE]
+Consider taking a snapshot of them before deleting.
```

Comment:
“Consider taking a snapshot before deleting” - by using the snapshot and revert tool? https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/17.1/html/backing_up_and_restoring_the_undercloud_and_control_plane_nodes/assembly_snapshot-and-revert-appendix_snapshot-and-revert-appendix

Reply:
@bogdando please correct me if needed, but i think it would be via something like: https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html-single/storage/index#lvms-about-volume-snapsot_logical-volume-manager-storage

Reply:
right, any backend for CSI snapshotter should work